### PR TITLE
Initial Scaffolding and Improving Full Calendar

### PR DESCRIPTION
**Pull request by Jake-Mitchell**
- adds Full Calendar, React Calendar packages
- sets up Full Calendar with month view, week view, and an input field to create new events

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
...
+
+# Business requirements
+
+- Calendar functionality needs to show day/month/week views
+
+## react-calendar
+
+- It appears the view available are only; "month", "year", "decade" or "century".
+
+## full-calendar
```
Comment (Jake-Mitchell) [inline README.md:11]: add full calendar info about different views

File: react-calendar/src/App.css [hunk 2]
```diff
@@ -1,38 +1,11 @@
-.App {
-  text-align: center;
+.container {
+  width: 100vw;
+  height: 100vh;
+  background-color: red;
```
Comment (Jake-Mitchell) [inline react-calendar/src/App.css:4]: alphebetization

File: react-calendar/src/App.css [hunk 3]
```diff
@@ -1,38 +1,11 @@
...
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.calendar {
+  width: 100%;
```
Comment (Jake-Mitchell) [inline react-calendar/src/App.css:8]: oooohhh

File: react-calendar/src/Calendar.css [hunk 4]
```diff
@@ -0,0 +1,117 @@
+.react-calendar {
+  width: 350px;
```
Comment (Jake-Mitchell) [inline react-calendar/src/Calendar.css:2]: can do a quick alphebetization

**Review by angepol — APPROVED** [review]
Looks good.

**Pull request merged**